packaging/arch/PKGBUILD

```diff
@@ -36,23 +42,35 @@ prepare() {
   # above describes.
   mkdir -p "$(dirname "$GOPATH/src/${_gourl}")"
   ln --no-target-directory -fs "$srcdir/$pkgname" "$GOPATH/src/${_gourl}"
+  # Patch snap-seccomp build flags not to link libseccomp statically.
+  sed -i -e 's/-Wl,-Bstatic -lseccomp -Wl,-Bdynamic/-lseccomp/' "$srcdir/$pkgname/cmd/snap-seccomp/main.go"
```

Comment:
I need to explore this because ideally there'd be no patch necessary. If we can use some conditional hacks just fur Ubuntu 16.04 and not touch anything else I would have much less gray hair about this. For now it's easier to use sed rather than (potentially) redo the patch each time.

Once @mvo5 is back I'd like to simplify this and move the special casing to ubuntu packging.